syntax = "proto3";

package heyp.proto;

option go_package = "github.com/uluyol/heyp-agents/go/pb";

message TestCompareMetrics {
  message Metric {
    string name = 1;
    double value = 2;
  }

  repeated Metric metrics = 1;
}
